src/color.rs

```diff
@@ -532,6 +539,13 @@ impl NumberOrPercentage {
             NumberOrPercentage::Percentage { unit_value } => unit_value,
         }
     }
+
+    fn value(&self, percentage_max: f32) -> f32 {
```

Comment:
I think the common name for this would be `percentage_basis`. But this is fine too.

Reply:
Cool, wasn't sure what to name it :smile: